plugin/ocgrpc/client_metrics.go

```diff
@@ -70,6 +71,14 @@ var (
 		Aggregation: view.Count(),
 	}
 
+	ClientStartedRPCsView = &view.View{
+		Measure: ClientStartedRPCs,
+		Name: "grpc.io/client/started_rpcs",
+		Description: "Count of started RPCs by method.",
```

Comment:
```suggestion
		Description: "Number of started client RPCs",
```

Reply:
Consistent with https://github.com/census-instrumentation/opencensus-java/blob/2e90f499881f601bfee1942b8bbfa2f79eaccc34/contrib/grpc_metrics/src/main/java/io/opencensus/contrib/grpc/metrics/RpcViewConstants.java#L511

Reply:
Done. Kept the period though to keep it consistent with descriptions around the file.

Reply:
still needs to be updated.